### ISSUE TITLE
Why your generated images' faces are not good? And the generated images' faces are not from those in your paper?

### ISSUE THREAD

**Issue by Xuzhenhao186** [YanzuoLu/CFLD#6]
Hello, I have a big question that why are your generated images' faces not good? And the generated images' faces are not from those in your paper?
Seeing your public generated images from Google Drive, I find that all the faces are not good, why is it? Why are the faces in your paper nice?
Thanks so much for your answer.

**Comment by YanzuoLu** [comment]
No. I'm not sure why you thought "all the faces are not good". That wouldn't be a correct statement. All of our qualitative results are coming from those in Google drive. You should be able to locate them as we have checked.

But we do have noticed some of the generated eyes on people's faces are weird to be white. 
I'm glad to discuss this and share my thoughts with you. I guess there are two reasons for this.

First, the model capacity of SD15 (mostly frozen and partially fine-tuning only) is not enough to generate high-quality faces. After all, faces often only occupy a small part of the image (compared to the face-specific diffusion model).
Second, as training progresses, quantitative indicators may be inaccurate and deceptive. Although the indicators are getting better, it may not be the main body of the character that is getting better but the background part.

In summary, we plan to make improvements in these two directions. One is to replace the foundation model with a more powerful one (SD2.1 in concurrent work PCDMs [ICLR 2024] or SDXL to support different aspect ratios), and the second is to shorten the training time, and it may be more reasonable to use the segmentation model to calculate quantitative indicators of the corresponding positions of the human body.

Thanks for you attention to our work!

**Comment by Xuzhenhao186** [comment]
I got it. Thank you so much!
